### PR TITLE
docs: FAQ

### PR DESCRIPTION
**Pull request by Arvindthiru**
### Description of your changes

Fixes #

I have:

- [x] Run `make reviewable` to ensure this PR is ready for review.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+## What are Fleet-owned Resources on the Hub and Member Clusters, and Can These Fleet-owned Resources be Modified by the User?
+
+- **Namespaces:**
+    - `fleet-system` (Hub Cluster)
+    - `fleet-member-{clusterName}` (Hub Cluster)
+- **Custom Resource Definitions**
+- **Custom Resources:**
+    - `MemberCluster` (Hub Cluster, user-facing resource)
+    - `InternalMemberCluster` (Hub Cluster, internal resource)
```
Comment (zhiying-lin) [inline docs/faq.md:11]: Should we use the CRD link here? you just need to point out which are user-facing resource and others are owned by fleet.

https://github.com/Azure/fleet/tree/main/charts/hub-agent/crdbases
https://github.com/Azure/fleet/tree/main/charts/member-agent/crdbases

same to fleet-networking

File: docs/faq.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+If there is a conflict because a particular resource already exists on the member cluster, the apply fails when attempting to propagate such a resource from the hub cluster.
+
+## What Happens If Users Change/Delete Resources That Were Placed from Hub to Member Clusters?
+
+Possible scenarios:
+
+- If the user updates the resource on the hub cluster, the update is propagated to all member clusters where the resource exists.
+- If the user deletes the resource on the hub cluster, the resource is deleted on all clusters to which it was propagated.
+- If the user modifies the resource on the member cluster, no automatic action occurs as it's a user-made modification.
```
[the remark this answers is not in the document]
Comment (Arvindthiru) [reply docs/faq.md:50]: I checked user modifications on the member cluster are not reconciled

[the remark this answers is not in the document]
Comment (Arvindthiru) [reply docs/faq.md]: After verifying, it looks like updates are ignored but deletions are reconciled back updated doc accordingly

File: docs/faq.md [hunk 3]
```diff
@@ -1,50 +1,68 @@
...
+
+### Fleet Networking member cluster resources:
+| Resource              | Resource Type        |
+|-----------------------|----------------------|
+| `MultiClusterService` | user-facing resource |
+| `ServiceExport`       | user-facing resource |
+
+**Fleet Networking APIs** are defined [here](https://github.com/Azure/fleet-networking/tree/main/api/v1alpha1), **Fleet Networking CRDs** are defined [here](https://github.com/Azure/fleet-networking/tree/main/config/crd/bases).
+
+`CRUD` operations are permitted for all the  `user-facing` resources, while `CRUD` operations are restricted for `internal` resources.  A majority of these `internal` resources are safeguarded by a series of validating webhooks, serving as a preventive measure to restrict users from making modifications to them
```
Comment (zhiying-lin) [inline docs/faq.md:43]: Can you move these statements before listing the CRDs?

File: docs/faq.md [hunk 4]
```diff
@@ -2,46 +2,38 @@
...
-| `ClusterResourceBinding`          | internal resource    |
-
-### Fleet member cluster resource:
-| Resource                | Resource Type     |
-|-------------------------|-------------------|
-| `InternalMemberCluster` | internal resource |
-| `AppliedWork`           | internal resource |
+Majority of the `internal` resources and fleet reserved namespaces described below are safeguarded by a series of validating webhooks, serving as a preventive measure to restrict users from making modifications to them.
+
+> The fleet reserved namespace are `fleet-system`, `fleet-member-{clusterName}` where clusterName is the name of each member cluster that has joined the fleet.
```
Comment (zhiying-lin) [inline docs/faq.md:7]:
```suggestion
The fleet reserved namespace are `fleet-system`, `fleet-member-{clusterName}` where clusterName is the name of each member cluster that has joined the fleet.
```

**Review by zhiying-lin — APPROVED** [review]

**Pull request merged**